tests/integration/test_checker.py

```diff
@@ -86,3 +92,50 @@ def test_parse_valid_run_time(self):
             isfile.return_value = True
             result = checker.parse_run_time('log')
         self.assertEqual(1, result)
+
+
+class TestCompletion(unittest.TestCase):
```

Comment:
you are missing tests for when individual steps fail

Reply:
I've added some more tests